data/src/main/java/org/oppia/data/backends/gae/NetworkSettings.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.data.backends.gae
+
+object NetworkSettings{
```

Comment:
Nit: need space before ``{``

Ditto for other files in this PR.

Reply:
Solved